src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedStaticConstructor.cs

```diff
@@ -371,5 +371,56 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
             var containingType = (SourceMemberContainerTypeSymbol)this.ContainingType;
             return containingType.CalculateSyntaxOffsetInSynthesizedConstructor(localPosition, localTree, isStatic: true);
         }
+
+#nullable enable
+        private ThreeState _shouldEmit = ThreeState.Unknown;
```

Comment:
>private ThreeState _shouldEmit = ThreeState.Unknown; [](start = 8, length = 52)

Please place this field declaration at the top of the class. We usually place them before a constructor. This makes it easier to see in a quick glance what data we already have, etc.